engines/bastion_katello/lib/bastion_katello/engine.rb

```diff
@@ -34,8 +34,8 @@ class Engine < ::Rails::Engine
         :config_generator =>  lambda do
           { 'consumerCertRPM' => consumer_cert_rpm,
             'defaultDownloadPolicy' => !Foreman.in_rake? && db_migrated && Setting['default_download_policy'],
-            'remoteExecutionPresent' => ::Katello.with_remote_execution?,
-            'hostToolingEnabled' => ::Katello.with_remote_execution?
+            'remoteExecutionPresent' => true,
+            'hostToolingEnabled' => true
```

Comment:
You didn't want to pull the trigger and also drop `remoteExecutionPresent` in the frontend? I was interested so wrote a5337ed2f7ec3255c9bf466f67fe1ab7845214ec. Looks like a nice cleanup.

Reply:
Yeah, it felt like a big can of worms. And I was still hoping for that angular removal

Reply:
You're being unusually positive with your hope.